src/JabbR/Views/Home/index.cshtml

```diff
@@ -25,6 +25,7 @@
     <link href="~/lib/highlightjs/github.css" rel="stylesheet">
     <link href="~/lib-checkin/css/KeyTips.css" rel="stylesheet">
     <link href="~/lib-checkin/css/emoji20.css" rel="stylesheet">
+<<<<<<< HEAD
```

Comment:
haha, yea something went wonky with a rebase and I didn't notice :hear_no_evil:  It will be fixed, promise :see_no_evil: